spec/helpers/list_helper_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe ListHelper, :type => :helper do
+  describe "#calculate_gravity" do
+    it "returns correct gravity _i dont know if this is correct" do
+      @total_gravity = 0
+      @recipe = FactoryGirl.create(:recipe)
+
+      ingredient =Ingredient.create(name:'Vienna Malt', type_class: 'Grain',
+       extract_potential: 1.036, alpha_acid: nil, attenuation: nil)
+
+      @recipe.lists.new({"ingredient_id"=>ingredient.id, "amount"=> "1.0", "boil_time"=>"0"})
```

Comment:
Line is too long. [93/80]<br>Redundant curly braces around a hash parameter.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.